docs/src/examples.md

```diff
@@ -0,0 +1,139 @@
+  ⋮  │          ⋮                ⋮          ⋮
+  18 │ 2018-09-10T17:00:00         21.2      56.8
+  19 │ 2018-09-10T18:00:00         20.0      60.3
+  20 │ 2018-09-10T19:00:00         18.4      64.8
+  21 │ 2018-09-10T20:00:00         16.7      70.8
+  22 │ 2018-09-10T21:00:00         15.0      77.3
+  23 │ 2018-09-10T22:00:00         13.9      83.1
+  24 │ 2018-09-10T23:00:00         12.7      87.0
+                                    9 rows omitted
+```
+
+We want to create some data features based on the time of day.
+One way to do this is with the `Periodic` transform, specifying a period of 1 day:
+
+```jldoctest example
+julia> periodic = Periodic(sin, Day(1));
+
+julia> df.hour_of_day_sin = FeatureTransforms.apply(df, periodic; cols=:time);
+
+julia> feature_df = df
+24×4 DataFrame
+ Row │ time                 temperature  humidity  hour_of_day_sin
+     │ DateTime             Float64      Float64   Float64
+─────┼─────────────────────────────────────────────────────────────
+   1 │ 2018-09-10T00:00:00         10.6      93.8         0.0
+   2 │ 2018-09-10T01:00:00          9.5      96.1         0.258819
+   3 │ 2018-09-10T02:00:00          8.9      94.8         0.5
+   4 │ 2018-09-10T03:00:00          8.9      92.4         0.707107
+   5 │ 2018-09-10T04:00:00          8.4      92.7         0.866025
+   6 │ 2018-09-10T05:00:00          8.4      97.3         0.965926
+   7 │ 2018-09-10T06:00:00          7.7     100.2         1.0
+   8 │ 2018-09-10T07:00:00          8.9      96.2         0.965926
+  ⋮  │          ⋮                ⋮          ⋮             ⋮
+  18 │ 2018-09-10T17:00:00         21.2      56.8        -0.965926
+  19 │ 2018-09-10T18:00:00         20.0      60.3        -1.0
+  20 │ 2018-09-10T19:00:00         18.4      64.8        -0.965926
+  21 │ 2018-09-10T20:00:00         16.7      70.8        -0.866025
+  22 │ 2018-09-10T21:00:00         15.0      77.3        -0.707107
+  23 │ 2018-09-10T22:00:00         13.9      83.1        -0.5
+  24 │ 2018-09-10T23:00:00         12.7      87.0        -0.258819
+                                                     9 rows omitted
+```
+
+Now suppose we want to use the first 22 hours as training data and the last 2 hours as test data.
+Our input features are the temperature, humidity, and periodic encodings for the current hour, and the outputs to predict are the temperature and humidity for the next hour. 
+
+```jldoctest example
+julia> train_df = feature_df[1:end-2, :];
+
+julia> test_df = feature_df[end-1:end, :];
+
+julia> input_cols = [:hour_of_day_sin, :temperature, :humidity];
+
+julia> output_cols = [:temperature, :humidity];
+```
+
+For many models it is helpful to normalize the training data.
+We can use `MeanStdScaling` for that purpose.
+Note that we are mutating the data frame in-place using `apply!`, and the order of columns specified does not matter.
+
+```jldoctest example
+julia> scaling = MeanStdScaling(train_df; cols=input_cols);
```

Comment:
Isn't it kinda weird to normalise a periodic signal? since it won't be normally distributed and the original Periodic transform already "scaled" it.

If you think so too, maybe just scale the `output_cols`?

Reply:
It doesn't matter much here because the periodic is roughly the same range of values anyway, but I'll just explain my view on normalising.

My logic is that all features should be normalised consistently, so that values lie within a similar small range around zero. In turn, model parameters require smaller updates. I think that mostly matters for neural nets with gradient descent. I'm not thinking as much about what the distribution of the feature is.